docs/components/connectors/out-of-the-box-connectors/uipath.md

```diff
@@ -0,0 +1,260 @@
+      "x-xss-protection":"1",
+      "x-content-type-options":"nosniff",
+      "x-robots-tag":"noindex,nofollow",
+      "content-type":"application/json; odata.metadata=minimal; odata.streaming=true",
+      "connection":"keep-alive",
+      "cache-control":"no-cache, no-store, must-revalidate",
+      "odata-version":"4.0"
+   },
+   "body":{
+      "@odata.context":"https://cloud.uipath.com/MyOrg/MyTenant/orchestrator_/odata/$metadata#QueueItems",
+      "@odata.count":1,
+      "value":[
+         {
+            "QueueDefinitionId":165001,
+            "Encrypted":false,
+            "Status":"New",
+            "ReviewStatus":"None",
+            "Key":"2196eb07-c96a-4f47-a734-326dd5d58a9d",
+            "Reference":"test",
+            "Priority":"Low",
+            "DeferDate":"2023-01-12T00:00:00Z",
+            "SecondsInPreviousAttempts":0,
+            "RetryNumber":0,
+            "SpecificData":"{\"DynamicProperties\":{\"test\":\"test\"}}",
+            "CreationTime":"2023-01-20T10:13:20.66Z",
+            "RowVersion":"AAAAAE2f4GY=",
+            "OrganizationUnitId":1964413,
+            "OrganizationUnitFullyQualifiedName":"MyCorporateWorkspace",
+            "Id":436141352,
+            "SpecificContent":{
+               "test":"test"
+            }
+         }
+      ]
+   }
+}
+```
+
+## Appendix
+
+### Using UIPath Connector best practice
+
+There is no guarantee a queue item will be processed right away. In that case, we suggest utilizing a BPMN polling loop pattern.
+
+:::note
+To avoid performance issues, it is recommended to limit the number of loop retries.
```

Comment:
Can we also link to details on this? Do you feel it is needed?

Reply:
Hi @bastiankoerber, @MaxTru 

Do you have handy any Camunda resource explaining loop retries best practices?
Maybe we should have a separate how-to, guideline, or example?

Reply:
I don't think we have an appropriate one.

I would try asking in [#ask-conslting](https://camunda.slack.com/archives/C01Q05D6YUU)

Reply:
Hi @christinaausley,

There is no explicit description for the polling loop pattern, so I did a bit of rewording.